openlibrary/templates/lib/nav_foot.html

```diff
@@ -47,6 +47,17 @@ <h2>$:_('Help')</h2>
           <a class="footer-icons__github" href="https://github.com/internetarchive/openlibrary">github</a>
         </aside>
       </div>
+      <div id="footer-locale-menu">
+        <h2>$:_('Change Website Language')</h2> 
+        <ul id="locale-options">
+          <li id="cs"><a href="/" title="$_('Czech')">Čeština (cs)</a> </li>
+          <li id="de"><a href="/" title="$_('German')">Deutsch (de)</a> </li>
+          <li id="en"><a href="/" title="$_('English')">English (en)</a></li>
+          <li id="es"><a href="/" title="$_('Spanish')">Español (es)</a></li>
+          <li id="fr"><a href="/" title="$_('French')">Français (fr)</a></li>
+          <li id="te"><a href="/" title="$_('Telugu')">Telugu (te)</a></li>
```

Comment:
I don't understand why the title is being translated, but the actual text isn't. Is that an oversight? Actually, since they should all be autoglottonyms, they will be the same in every language, so don't need to be included in the translation files.

Reply:
[My understanding](https://github.com/internetarchive/openlibrary/tree/master/openlibrary/i18n#updating-an-existing-language) was that any new string provided should be included in the default `messages.pot` file but it is acceptable to not commit the translated text if one's unsure about it.

So I added the title as well as the Czech and Telugu strings (the rest of the languages were already in the `messages.pot` file) but didn't provide their translated counterparts as I were unsure about their validity.

Should I completely remove them from the `messages.pot` file?

Reply:
Yes, I think you can use the same string for both the text and the title e.g.
```
          <li id="cs"><a href="/" title="Čeština">Čeština (cs)</a> </li>
```
It doesn't need to be translated, because users will be expecting to find it in their native language ie not "Czech," which they might not recognize if they don't speak English.